prepare-vms/README.md

```diff
@@ -1,74 +1,73 @@
 # Trainer tools to prepare VMs for Docker workshops
 
-There are several options for using these tools:
+## 1. Prerequisites
 
-### Clone the repo
+* [Docker](https://docs.docker.com/engine/installation/)
+* [Docker Compose](https://docs.docker.com/compose/install/)
 
-    $ git clone https://github.com/soulshake/prepare-vms.git
-    $ cd prepare-vms
-    $ docker-compose build
-    $ mkdir $HOME/bin && ln -s `pwd`/trainer $HOME/bin/trainer
-
-### Via the image
-
-    $ docker pull soulshake/prepare-vms
-
-### Submodule
+## 2. Clone the repo
 
-This repo can be added as a submodule in the repo of the Docker workshop:
-
-    $ git submodule add https://github.com/soulshake/prepare-vms.git
-
-## Setup
+    $ git clone https://github.com/jpetazzo/orchestration-workshop.git
+    $ cd orchestration-workshop/prepare-vms
+    $ docker-compose build
+    $ ./trainer <commands>  # See "Summary of commands" section below
 
-### Export needed envvars
+## 3. Preparing the environment
 
 Required environment variables:
 
 * `AWS_ACCESS_KEY_ID`
 * `AWS_SECRET_ACCESS_KEY`
 * `AWS_DEFAULT_REGION`
 
-
-
-### Update settings.yaml
+### 4. Update settings.yaml
 
 If you have more than one workshop:
 
-    $ cp settings.yaml settings/YOUR_WORKSHOP_NAME-settings.yaml
-    $ ln -s settings/YOUR_WORKSHOP_NAME-settings.yaml `pwd`/settings.yaml
+    $ cp settings/settings.yaml settings/YOUR_WORKSHOP_NAME-settings.yaml
```

Comment:
Normally it's not necessary anymore to `cp` the settings file around, since it's passed as a parameter to `deploy` and `cards`. WDYT?